opencensus/common/resource.py

```diff
@@ -49,22 +49,32 @@
 _UNQUOTE_RE = re.compile(r'^([\'"]?)([^\1]*)(\1)$')
 
 
-def merge_resources(r1, r2):
-    """Merge two resources to get a new resource.
+def merge_resources(resource_list):
```

Comment:
@c24t I'd like to hear about your opinion.

Shall we consider the signature of `os.path.join`? https://docs.python.org/2/library/os.path.html#os.path.join

Reply:
Part of the motivation for making the arg a list was to match the [signature in java](https://github.com/c24t/opencensus-java/blob/c0f3a4cf4c437b9a4ac983f1bd44781f60d0182f/api/src/main/java/io/opencensus/resource/Resource.java#L124), but since `*args` in python is more idiomatic than varargs in java, maybe this is the better approach here, and I like this pattern as a user.

You think callers are more likely to have to make a list to call `merge_resources` than have to splat a list if we mimic `os.path.join`?

Reply:
I put up https://github.com/c24t/opencensus-python/pull/1/files#diff-21856523a57af46b9f02f815944713e3R89 to see what this would look like.

Reply:
I don't have strong opinion here, just putting another option and see which one you would prefer.
Please do what you think is better. Either way looks good to me :)